modules/currency-l0/src/main/scala/org/tessellation/currency/l0/snapshot/CurrencySnapshotConsensusFunctions.scala

```diff
@@ -61,7 +71,11 @@ object CurrencySnapshotConsensusFunctions {
       events: Set[CurrencySnapshotEvent]
     ): F[(CurrencySnapshotArtifact, CurrencySnapshotContext, Set[CurrencySnapshotEvent])] = {
 
-      val blocksForAcceptance = events
+      val (blocks: List[Signed[CurrencyBlock]], dataBlocks: List[Signed[DataApplicationBlock]]) = events.filter { event =>
+        if (maybeDataApplicationService.isEmpty) event.isLeft else true
```

Comment:
I think it should be possible to write the filter without returning `true`

Reply:
I've copied as we have it in the global l0 consensus. any idea how to change that?

Reply:
changed to use `collect`